src/plugins/inspectorViews/styles/StylesInspectorViewProvider.js

```diff
@@ -47,27 +64,42 @@ export default function StylesInspectorViewProvider(openmct) {
     canView: function (selection) {
       const objectSelection = selection?.[0];
       const objectContext = objectSelection?.[0]?.context;
-      const layoutItem = objectContext?.layoutItem;
       const domainObject = objectContext?.item;
+      const hasStyles = domainObject?.configuration?.objectStyles;
       const isFlexibleLayoutContainer =
         domainObject?.type === 'flexible-layout' && objectContext.type === 'container';
+      const isLayoutItem = objectContext?.layoutItem;
 
-      if (layoutItem) {
+      if (isLayoutItem || hasStyles) {
         return true;
       }
 
       if (!domainObject || isFlexibleLayoutContainer) {
         return false;
       }
 
-      const type = openmct.types.get(domainObject.type);
+      const typeObject = openmct.types.get(domainObject.type);
 
       return (
-        isLayoutObject(objectSelection, domainObject.type) || isCreatableObject(domainObject, type)
+        isLayoutObject(objectSelection, domainObject.type) ||
+        isCreatableObject(domainObject, typeObject)
       );
     },
     view: function (selection) {
       let _destroy = null;
+      const objectSelection = selection?.[0];
+      const objectContext = objectSelection?.[0]?.context;
+      const domainObject = objectContext?.item;
+      const onlyEditMode = [
```

Comment:
These are views where we _only_ want to show the style inspector in edit mode, is that correct? Why is that?

Reply:
My reason for trying to understand this is I'm concerned about the amount of plugin-specific code we are building into this view. I'd like to keep it as agnostic as possible, and have plugins opt-in to features like styling.

Reply:
I'll let @charlesh88 answer this one!

Reply:
@akhenry I think the original idea was reducing the number of tabs in view while in browse mode. That said, I agree with reducing complexity have no real objection to showing the Style tab in browse mode for those view as long as it's prioritized properly in that mode, which will be different than in edit mode.

So, this was the original spec for those views:
<img width="1484" alt="image" src="https://github.com/user-attachments/assets/a3271472-762b-444b-bf13-702806c89338" />

And here it is with updated priorities:
<img width="1484" alt="image" src="https://github.com/user-attachments/assets/85840ed8-64ef-49af-a77a-674b3c8e5f61" />

Reply:
I'll make the update!